travis/README.md

```diff
@@ -1,6 +1,14 @@
 # Zipkin release process
 
-The Zipkin release process is controlled by [publish.sh](publish.sh). Releases are initiated by pushing a tag `major.minor.revision` or `major.minor.revision-qualifier`, where `qualifier` is usually something like `rc3`. A high-level overview of how the release happens in each case:
+The Zipkin release process is controlled by [publish.sh](publish.sh).
+Releases are initiated by pushing a tag `major.minor.revision` or `major.minor.revision-qualifier`,
+where `qualifier` is usually something like `rc3`.
+
+There is currently no automated testing during the release process. As the person pushing a release tag, you're
```

Comment:
nit add link https://travis-ci.org/openzipkin/zipkin
nit s/version you're tagging/commit you're tagging/